src/clj/game/cards/events.clj

```diff
@@ -460,6 +460,29 @@
    "Easy Mark"
    {:msg "gain 3 [Credits]" :effect (effect (gain :credit 3))}
 
+   "Embezzle"
+   (letfn [(name-string [cards] (clojure.string/join " and " (map :title cards)))] ; either 'card' or 'card1 and card2'
+    {:req (req hq-runnable)
+     :effect (effect
+              (run :hq {:req (req (= target :hq))
+                        :replace-access
+                        {:mandatory true
+                         :msg (msg "reveal 2 cards from HQ and trash all " target "s") ;should maybe lower-case target
+                         :prompt "Choose a card type"
+                         ; Identity is included here because it's technically allowed and relevant with e.g. Hostile Infrasructure
+                         :choices ["Agenda" "Asset" "Upgrade" "Operation" "ICE" "Identity"] 
```

Comment:
Card only allows you to name "asset, ice, operation, or upgrade".

Reply:
Ah, that's a good point. I mentally just read that as 'a card type', thanks.